superflore/generators/ebuild/run.py

```diff
@@ -69,33 +71,31 @@ def main():
     global overlay
     global preserve_existing
     global mode
-    if len(sys.argv) == 2:
-        arg1 = sys.argv[1].replace('--', '')
-
-        if arg1 == 'all' or arg1 == 'update':
-            mode = arg1
-            preserve_existing = arg1 == 'all'
-        else:
-            preserve_existing = False
-            if arg1 not in active_distros:
-                ros_overlay.error('Unknown ros distro "{0}"'.format(arg1))
-                ros_overlay.error('Exiting...')
-                sys.exit(1)
-            else:
-                msg = 'Regenerating all packages for distro "{0}"'.format(arg1)
-                ros_overlay.info(msg)
-                mode = arg1
-    elif len(sys.argv) >= 2:
-        ros_overlay.error('Invalid arguments!')
-        print_usage()
+
+    parser = argparse.ArgumentParser(
+        'Deploy ROS packages into Gentoo Linux.')
+    parser.add_argument(
+        '--distro', help='regenerate packages for the specified distro.', type=str)
```

Comment:
It would be better to use a default value here instead of putting the logic below. It's better for visibility as it will appear in the help. https://github.com/allenh1/superflore/pull/39/files#diff-5a7657ae13022a44815efc96706d2233R84